src/patternfly/components/ChipGroup/chip-group.scss

```diff
@@ -8,7 +8,7 @@
   --pf-c-chip-group--PaddingBottom: 0;
   --pf-c-chip-group--PaddingLeft: 0;
   --pf-c-chip-group--RowGap: var(--pf-global--spacer--sm);
-  --pf-c-chip-group--ColumnGap: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--ColumnGap: 0;
```

Comment:
it's fine to leave, but this isn't used, so we could remove it, too.

Reply:
I think for flexibility it's good to have (it is used on line 43 if that's what you meant)